<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffPage</class>
 <widget class="QMainWindow" name="StaffPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1051</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1051</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HomePageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home Page</string>
    </property>
   </widget>
   <widget class="QLabel" name="AceTrainingTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AceTraining</string>
    </property>
   </widget>
   <widget class="QLabel" name="WelcomeMessage">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome Back, [Insert Name]</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to your Home Page, on this page you will 
be able to view your information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignOutButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CoursesButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AssessmentsButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Assessments</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DegreeButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Degrees</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateOfBirthLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date of Birth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GenderLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CountyLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>County:</string>
    </property>
   </widget>
   <widget class="QLabel" name="AddressLine1Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Address Line</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GenderBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PostcodeBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="EmailAddressLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="StudentFirstNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First Name: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>630</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CountyBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateOfBirthBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>480</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstNameBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="StudentSurnameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Surname:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NationalityBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddressLine1Box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PostcodeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Postcode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NationalityLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nationality:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SurnameBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelNino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>National Insurance Number (click to show)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NationalityLabel_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Job Role</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BoxRole">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CountyLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Office Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BoxSalary">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="BoxNINO">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SalaryLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Annual Salary (click to show)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BoxOfficeNum">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnShowNINO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnShowSalary">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnBackup">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRestore">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
